packages/btc/src/query/source.ts

```diff
@@ -23,7 +43,7 @@ export class DataSource {
         const bOrder = `${b.status.block_height}${b.vout}`;
         return Number(aOrder) - Number(bOrder);
       })
-      .map((row): UnspentOutput => {
+      .map((row): Utxo => {
         return {
```

Comment:
let's say:
- a.status.block_height = 100
- b.status.block_height = 20
- a.vout = b.vout = 0

"1000" < "200" === true

Reply:
Those stirngs will be converted into numbers before comparing, so actually it's like this: `Number('1000') < Number('200') === false`. But you reminded me that there will be issues if the digit difference between `a.vout` and `b.vout`, like: "100|0" vs "20|20".